src/components/Calculator.js

```diff
@@ -0,0 +1,50 @@
+import react from 'react';
+import '../App.css';
+
+class Calculator extends react.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calculator">
+        <div className="row">
+          <input className="input" value="0" />
+        </div>
+        <div className="row">
+          <p className="colr-grey item">AC</p>
```

Comment:
Please ensure that the `Calculator.js` component should match the following given design.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/163880890-2cfc3faa-619a-400c-b331-1e9dc9b0395a.png) | ![image](https://user-images.githubusercontent.com/10332499/163880944-b54e4686-ab56-4a33-b8f7-6e0615297c68.png)

Reply:
Thanks solved